### PR TITLE
cut release v1.2.0

### LINKED ISSUE #172: [Task] create release v1.2.0

### PR DESCRIPTION
**Pull request by gbraad**
fixes #172

### CODE REVIEW

**Review by anjannath — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "name": "openshift-local",
   "displayName": "Red Hat OpenShift Local",
   "description": "Integration for Red Hat OpenShift Local clusters",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (praveenkumar) [inline package.json:6]: 👍🏼

**Review by praveenkumar — APPROVED** [review]

**Comment by praveenkumar** [comment]
Build is failing on mac os.

**Comment by gbraad** [comment]
### Release links
  * `ghcr.io/crc-org/crc-extension:v1.2.0`
  * `quay.io/redhat-developer/openshift-local-extension:v1.2.0`
  * https://github.com/crc-org/crc-extension/releases/tag/v1.2.0

**Comment by gbraad** [comment]
@jeffmaury What are the next steps to make sure PD can pick this up?

**Comment by jeffmaury** [comment]
Catalog has been updated: https://github.com/containers/podman-desktop-catalog/commit/e95145a162cbabe1b08e54a081615508a6a2ecc5

**Pull request merged**